AEPIntegrationTests/IdentityIntegrationTests.swift

```diff
@@ -44,20 +45,32 @@ class IdentityIntegrationTests: XCTestCase {
             unregisterExpectation.fulfill()
         }
 
+        // Used for event related assertions
+        MobileCore.unregisterExtension(InstrumentedExtension.self) {
+            unregisterExpectation.fulfill()
+        }
+
         wait(for: [unregisterExpectation], timeout: 3)
-        EventHub.shared.shutdown()
 
+        EventHub.shared.shutdown()
         ServiceProvider.shared.reset()
         EventHub.reset()
     }
 
     func initExtensionsAndWait() {
         let initExpectation = XCTestExpectation(description: "init extensions")
         MobileCore.setLogLevel(.trace)
-        MobileCore.registerExtensions([Identity.self, Lifecycle.self, Signal.self]) {
+        MobileCore.registerExtensions([InstrumentedExtension.self, Identity.self, Lifecycle.self, Signal.self]) {
             initExpectation.fulfill()
         }
         wait(for: [initExpectation], timeout: 1)
+
+        // Add an additional wait time
+        let semaphore = DispatchSemaphore(value: 0)
```

Comment:
Is there a valid reason for adding additional wait time? If so, can you also add it to the comment?

Reply:
Can you use `XCTWaiter.wait(for: [some_expectation], timeout: 0.5)` instead of semaphore?